src/rawtoaces_util/image_converter.cpp

```diff
@@ -801,27 +869,21 @@ bool ImageConverter::parse_parameters( const OIIO::ArgParse &arg_parser )
     if ( arg_parser["list-cameras"].get<int>() )
     {
         auto cameras = supported_cameras();
-        std::cout << std::endl
-                  << "Spectral sensitivity data are available for the "
-                  << "following cameras:" << std::endl;
-        for ( const auto &camera: cameras )
-        {
-            std::cerr << std::endl << camera;
-        }
-        std::cerr << std::endl;
+        std::cout
+            << std::endl
+            << "Spectral sensitivity data is available for the following cameras:"
+            << std::endl
+            << OIIO::Strutil::join( cameras, "\n" ) << std::endl;
+        exit( 0 );
     }
 
     if ( arg_parser["list-illuminants"].get<int>() )
     {
-        // gather a list of illuminants supported
         auto illuminants = supported_illuminants();
-        std::cerr << std::endl
-                  << "The following illuminants are supported:" << std::endl;
-        for ( const auto &illuminant: illuminants )
-        {
-            std::cerr << std::endl << illuminant;
-        }
-        std::cerr << std::endl;
+        std::cout << std::endl
+                  << "The following illuminants are supported:" << std::endl
+                  << OIIO::Strutil::join( illuminants, "\n" ) << std::endl;
+        exit( 0 );
```

Comment:
fixes the problem of showing help message for these two commands. Returning `false` would cause for programm to exit with `1` in main, shich semanticallt incorrect.